src/srr-mapping.ts

```diff
@@ -740,3 +770,17 @@ export function handleProvenanceDateMigrationFix(
   prov.createdAt = event.params.originTimestamp;
   prov.save();
 }
+
+export function handleLockExternalTransfer(event: LockExternalTransferEvent): void {
+  logInvocation("handleLockExternalTransfer", event);
+  let srrId = event.params.tokenId.toString();
+  let srr = SRR.load(srrId);
+  if(srr == null){
+    log.error("received lock external transfer event but srr not found: {}", [
+      srrId,
+    ]);
+    return;
```

Comment:
I'm not very sure but we want to log error and `return` instead of `throw` here?